# one input, one output

string map_id
---
string message
